css/index.css

```
@@ -256,6 +256,75 @@ a:-webkit-any-link {
   align-self: center;
 }
 
+.message-form {
+  background: var(--primary-color);
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+  width: 100%;
+  max-width: 500px;
+  box-sizing: border-box;
+}
+
+.message-form label {
+  display: block;
+  margin-bottom: 8px;
+  font-weight: bold;
+}
+
+.message-form input,
+.message-form textarea {
+  width: calc(100% - 20px);
+  padding: 10px;
+  margin-bottom: 12px;
+  border: 1px solid #ddd;
+  border-radius: 4px;
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+.message-form textarea {
+  height: 150px;
+  resize: vertical;
+}
+
+.message-form button {
+  background-color: #22D4FD;
+  color: #fff;
+  border: none;
+  padding: 10px 15px;
+  font-size: 16px;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: background-color 0.3s ease;
+}
+
+.message-form button:hover {
+  background-color: #0056b3;
+}
+
+#meesage-section {
+  margin-top: 20px;
+  width: 100%;
+  max-width: 500px;
+}
+
+#meesage-section h2 {
```

You have the same typo here, again.

```suggestion
#message-section h2 {
```